src/stores/attributes-product.md

```diff
@@ -2,152 +2,60 @@
 title: Product Attributes
 ---
 
-Product attribute properties determine how an attribute can be used in the catalog, its appearance and behavior in the store, and in data transfer operations. The properties and labels associated with each attribute determine how it can be used, and its presentation in the store.
+Product attribute properties determine how an attribute can be used in the catalog, its appearance and behavior in the store, and the data included in data transfer operations. The properties and labels associated with each attribute determine how it can be used and its presentation in the store.
```

Comment:
None of these tables seem to be rendering properly...at least in Github

Reply:
Wonder if that property-table attribute is breaking it...

Reply:
Verified this in staging build 234. The property-table attribute is probably the issue in the GitHub rendering, but necessary to fix the issue with auto sizing and some of these extra long field names.